CI/package-mudlet-for-windows.sh

```diff
@@ -126,11 +125,12 @@ ZIP_FILE_NAME="Mudlet-${MSYSTEM}"
 echo ""
 echo "Examining Mudlet application to identify other needed libraries..."
 NEEDED_LIBS=$("${MINGW_INTERNAL_BASE_DIR}/bin/ntldd" --recursive ./mudlet.exe \
-  | /usr/bin/grep -v "Qt5" \
+  | /usr/bin/grep -v "Qt6" \
   | /usr/bin/grep -i "mingw" \
   | /usr/bin/cut -d ">" -f2 \
   | /usr/bin/cut -d "(" -f1 \
-  | /usr/bin/sort)
+  | /usr/bin/sort \
+  | /usr/bin/uniq)
```

Comment:
I can't recall but it might be possible to get multiples from this pipeline - the `uniq` will take care of those...